.pre-commit-config.yaml

```diff
@@ -4,17 +4,19 @@ exclude: |
     ^.*/__init__\.py$|
     ^.*?/\.venv/.*$|
     ^.*?/node_modules/.*$|
-    
+
     ^.*?/charts/.*$|
     ^airbyte-integrations/bases/base-normalization/.*$|
     ^.*?/normalization_test_output/.*$|
-    
+
     ^.*?/pnpm-lock\.yaml$|
     ^.*?/source-amplitude/unit_tests/api_data/zipped\.json$|
-    
+
     # Generated/test files
     ^airbyte-ci/connectors/metadata_service/lib/metadata_service/models/generated/.*$|
     ^.*?/airbyte-ci/connectors/metadata_service/lib/tests/fixtures/.*/invalid/.*$|
+    ^airbyte-ci/connectors/metadata_service/lib/tests/fixtures/.*/invalid/.*$|
+    ^.*?/airbyte-ci/connectors/pipelines/tests/test_format/non_formatted_code/.*$|
```

Comment:
Formatters seem inconcistent on whether they will accept `^` or `^.*?/` at the start of the path - and this may also have something to do with the relative path or path that the file is cloned at. Adding both ways solves this in the case above that I found was inconcistently applied.

Reply:
Yeah but I think we can live with that. Ultimately I think we can remove the tests for those unformatted things because, well, those are FORMATTER TESTS